README.md

```diff
@@ -55,16 +56,16 @@ variables:
 #### Command to run:
 
   ```
-  $ python main.py 'url' page_count(optional)
+  $ python main.py 'url' page_count(optional) -enable_aimations 'on'(optional)
   ```
 
   Example:
 
+    $ python main.py 'https://www.scoopwhoop.com/entertainment/memes-from-dd-ramayan/'
 
     $ python main.py 'https://www.scoopwhoop.com/entertainment/memes-from-dd-ramayan/' 5
 
-    $ python main.py 'https://www.scoopwhoop.com/entertainment/memes-from-dd-ramayan/'
-
+    $ python main.py 'https://www.scoopwhoop.com/entertainment/memes-from-dd-ramayan/' -enable_animations 'on'
```

Comment:
Its better to have boolean here, instead of `on`

Reply:
Changed the options to yes/no.

Reply:
Please check this ocne:
https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse

Reply:
Updated accordingly!